Gemfile

```diff
@@ -23,6 +23,10 @@ end
 
 gem 'simplecov', '~> 0.8'
 
+if RUBY_VERSION < '2.0.0' || RUBY_ENGINE == 'java'
+  gem 'json', '< 2.0.0'
+end
```

Comment:
Ok I've looked into this some more, it's a dependency of simplecov. Which accepts `json (>= 1.8, < 3)`, but `json >= 2.0.0` requires Ruby 2.0.0, so it's either pin it for all, or pin it on older rubies.